src/applications/personalization/profile/components/personal-information/ContactInformationView.jsx

```diff
@@ -21,7 +21,15 @@ const ContactInformationView = props => {
   }
 
   if (fieldName === FIELD_NAMES.EMAIL) {
-    return <span>{data.emailAddress}</span>;
+    // add zero-width spaces after @ and . symbols so very long email addresses
+    // will wrap at those symbols if needed
+    const wrappableEmailAddress = data.emailAddress.replace(
```

Comment:
hah this is cool

Reply:
heh, but did you read about the problem it introduces?

Reply:
Yeah I did. Personally, I almost never copy and paste my email address because typing it is so fast. Agreed the edge case is small enough to not worry too much about.

Reply:
Try this:

The JS wraps the `@` and `.` in a span

```js
data.emailAddress
  .split(/(@|\.)/)
  .map(str => (/(@|\.)/.test(str) ? <span>{str}</span> : str))
```
then the CSS adds a zero-width white space after the span

```css
span:after {
  content: '\200B'
}
```

Here's a demo - https://plnkr.co/edit/nkcZMxcwx59GFQdM?preview

Reply:
TIL that this is a thing:

![IMG_0510](https://user-images.githubusercontent.com/20728956/125167686-a664e800-e156-11eb-92c4-97855183212e.jpeg)